docs/REST API/rest-api.md

```diff
@@ -1944,7 +1944,33 @@ If `after` is being used with `created_at` or `updated_at`, the table must be sp
       "display_text": "2ceca32fe484",
       "team_id": null,
       "team_name": null,
-      "pack_stats": null,
+      "pack_stats": [
+        {
+          "pack_id": 0,
+          "pack_name": "Global",
+          "type": "global",
+          "query_stats": [
+            {
+            "scheduled_query_name": "Get recently added or removed USB drives",
+            "scheduled_query_id": 5535,
+            "query_name": "Get recently added or removed USB drives",
+            "discard_data": false,
+            "last_fetched": null,
```

Comment:
Should we use `last_updated` to differentiate from the use of "fetch" as "last time this host responded"?

Reply:
as in 
<img width="252" alt="image" src="https://github.com/fleetdm/fleet/assets/61553566/574c23e6-88cd-49f2-8722-cee927c0e595">

Reply:
I say let's keep the API naming consistent with what's in the other endpoint, unless there's a conflict.